Makefile

```diff
@@ -42,18 +42,24 @@ build-all: dockerized-build-ui
 	$(MAKE) --directory=vmdk_plugin $@
 	$(MAKE) --directory=plugin all
 
+build-global: dockerized-build-ui
```

Comment:
"global" is very confusing. "Shared" is way better. Yes, I know all is shared on one VM, but it's easy to explain in comments